src/lib/connectTraceToPlot.js

```diff
@@ -90,7 +90,8 @@ export default function connectTraceToPlot(WrappedComponent) {
         fullData.forEach(t =>
           Object.keys(t).forEach(key =>
             setMultiValuedContainer(multiValuedFullContainer, t, key, {
-              searchArrays: true,
+              // don't search arrays if all traces are from splits
+              searchArrays: data.length > 1,
```

Comment:
no.. I don't think I agree with this..
if data.length === 1, that doesn't mean that all traces are from splits..
I think this pr is ok without this adjustment, and we can think about this one some more in another pr.